Samples/GDS/Client/Controls/ApplicationCertificateControl.cs

```diff
@@ -316,41 +313,48 @@ private async void CertificateRequestTimer_Tick(object sender, EventArgs e)
 
                     if (!String.IsNullOrEmpty(m_application.CertificateStorePath) && !String.IsNullOrEmpty(m_application.CertificateSubjectName))
                     {
-                        CertificateIdentifier cid = new CertificateIdentifier()
-                        {
+                        CertificateIdentifier cid = new CertificateIdentifier() {
                             StorePath = m_application.CertificateStorePath,
                             StoreType = CertificateStoreIdentifier.DetermineStoreType(m_application.CertificateStorePath),
                             SubjectName = m_application.CertificateSubjectName.Replace("localhost", Utils.GetHostName())
```

Comment:
there is a helper in Utils to replace the hostname

Reply:
is Utils.ReplaceDCLocalhost okay or better to use Utils.ReplaceLocalhost to catch uncommon SubjectNames?

Reply:
Utils.ReplaceDCLocalhost should be it